examples/conform.py

```diff
@@ -53,12 +54,14 @@
 
 def parse_args():
     """ parse arguments out of sys.argv """
-    parser = argparse.ArgumentParser(description=__doc__)
+    parser = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=RawTextHelpFormatter)
```

Comment:
interesting, I usually use the formatter with defaults.  The raw text one preserves the formatting in the file docstring block comment?

Reply:
Yes, the default formatter was removing all the line breaks.